lib/utils/h1_helper.ts

```diff
@@ -172,6 +175,57 @@ function processDC(decodeResult: DecodeResult, data: string[]): boolean {
     }
     return true;
 }
+function processWindData(decodeResult: DecodeResult, message: string): boolean {
+    if (decodeResult.raw.wind_data === undefined) {
+        decodeResult.raw.wind_data = [];
+    }
+    const flightLevel = Number(message.slice(0, 3));
+    const fields = message.slice(4).split('.'); // strip off altitude and comma
+    fields.forEach((field) => {
+        const data = field.split(',');
+        const waypoint = data[0];
+        const windData = data[1];
+        const windDirection = Number(windData.slice(0, 3));
+        const windSpeed = Number(windData.slice(3));
+
+        if (data.length === 3) {
+            const tempData = data[2];
+            const tempFlightLevel = Number(tempData.slice(0, 3));
```

Comment:
note that this usually matches `flightLevel`, sometimes it does not

Reply:
When it isn't matching flight level, what is it? Or is that still a mystery?

Reply:
I *thought* i found a message that had the temp at a lower flight level, but now i can't find it